core/server/data/migration/reset.js

```diff
@@ -12,11 +11,23 @@ var Promise      = require('bluebird'),
  * Deletes all the tables defined in the schema
  * Uses reverse order, which ensures that foreign keys are removed before the parent table
  *
+ * @TODO:
+ * - move to sephiroth
+ * - then deleting migrations table will make sense
+ *
  * @returns {Promise<*>}
  */
 reset = function reset() {
+    var result;
+
     return Promise.mapSeries(schemaTables, function (table) {
         return commands.deleteTable(table);
+    }).then(function (_result) {
+        result = _result;
+
+        return commands.deleteTable('migrations');
```

Comment:
I don't see a schema for a migrations table? I also don't remember a discussion about adding one - why do we need it?